genome_assembly/hic-scaffolding.md

```diff
@@ -0,0 +1,82 @@
+
+This workflow has been created from a Vertebrate Genomes Project (VGP) scaffolding workflow. 
+* For more information about the VGP project see the [Galaxy-VGP project page](https://galaxyproject.org/projects/vgp).
+* The VGP scaffolding workflow is hosted on [Dockstore](https://dockstore.org/workflows/github.com/iwc-workflows/Scaffolding-HiC-VGP8/main:main?tab=info). Please see that link for the workflow diagram and more information if required. 
+* Some minor changes have been made to better fit with TSI project data: optional inputs of SAK info and sequence graph have been removed; the required input format for the genome is changed from gfa to fasta; and the estimated genome size now requires user input rather than being extracted from output of a previous workflow.
+
+Please see the HiC Scaffolding section in the [VGP assembly tutorial](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html) for additional information about this workflow. 
+
+Note: If you initially assembled the genome with HiFi data only, and you have new HiC data, you may wish to consider re-assembling the genome with the [VGP HiFi-HiC assembly pipeline](https://dockstore.org/workflows/github.com/iwc-workflows/Assembly-Hifi-HiC-phasing-VGP4/main:main?tab=info), which can give better results than using HiFi data alone. 
+
+{% include callout.html type="note" content="If you need help, the Galaxy community is both approachable and helpful. [Ask them questions!](https://help.galaxyproject.org/)" %}
+
+### Register and login
+
+1. To register for Galaxy Australia, visit the [login page](https://usegalaxy.org.au/login).
+2. Click the ```Register here``` link.
+3. Complete the registration wizard and click ```Create```.
+4. Login to your account!
+
+### Upload data files
+
+Please see the how-to guide for HiFi genome assembly for additional information about uploading data from the Bioplatforms Australia Data Portal.
+
+* In Galaxy Australia, create a new history
+* Import data
+    * ```assembly.fasta```. This file may be in one of your Galaxy histories.
+    * HiC data: concatenated ```HiC_F.fastqsanger.gz```, concatenated ```HiC_R.fastqsanger.gz```
+  
+### Import the scaffolding workflow
+
+Please see the how-to guide for HiFi genome assembly for additional information about how to import and run workflows. A brief summary is given here:
+
+* In Galaxy, go to the Workflow tab
+* Click "Import"
+* Click on the tab at the right, "TRS ID"
+* Make sure the "TRS server" is set to `workflowhub.eu`
+* At "TRS ID", type in `1054`
+* Click on the version you wish to import (e.g. Version 1)
```

Comment:
This could be replaced with a TRS link, like the other guides: e.g. https://github.com/AustralianBioCommons/how-to-guides/blob/main/genome_annotation/Transcriptome.md#run-rna-seq-qc-and-trimming-workflow

This would make it much easier for a user to access the workflow.

Reply:
Yep, changed this.